README.md

```diff
@@ -148,6 +147,72 @@ token: xxxxxxxxxxxx # token of the telegram bot
 bot_tag: "@bot" # tag of the telegram bot
 ```
 
+### Settings override
+
+The settings may also be set through environment variables.  
+All the env vars with the same name (case insensitive) will override the ones in the settings file.
+To update the **meme** settings, prefix the env var name with **MEME\_**. The same is true for the **test** settings, that have to be prefixed with **TEST\_** and the **debug** settings, to be prefixed with **DEBUG\_**.
```

Comment:
Grammarly check suggests `which have` instead of `that have` but it should not matter.
```suggestion
To update the **meme** settings, prefix the env var name with **MEME\_**. The same is true for the **test** settings, which have to be prefixed with **TEST\_** and the **debug** settings, to be prefixed with **DEBUG\_**.
```